sdk/textanalytics/Azure.AI.TextAnalytics/tests/RecognizeEntitiesTests.cs

```diff
@@ -154,56 +126,48 @@ public void RecognizeEntitiesBatchWithInvalidDocumentBatch()
             Assert.AreEqual(TextAnalyticsErrorCode.InvalidDocumentBatch, ex.ErrorCode);
         }
 
+        [Test]
+        public async Task RecognizeEntitiesBatchConvenienceTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            var documents = s_batchConvenienceDocuments;
+
+            RecognizeEntitiesResultCollection results = await client.RecognizeEntitiesBatchAsync(documents);
+
+            ValidateBatchDocumentsResult(results);
+        }
+
         [Test]
         public async Task RecognizeEntitiesBatchConvenienceWithStatisticsTest()
         {
             TextAnalyticsClient client = GetClient();
-            var documents = batchConvenienceDocuments;
+            var documents = s_batchConvenienceDocuments;
 
             RecognizeEntitiesResultCollection results = await client.RecognizeEntitiesBatchAsync(documents, "en", new TextAnalyticsRequestOptions { IncludeStatistics = true });
 
-            foreach (RecognizeEntitiesResult result in results)
-            {
-                Assert.GreaterOrEqual(result.Entities.Count(), 1);
-            }
-
-            Assert.IsNotNull(results.Statistics.DocumentCount);
-            Assert.IsNotNull(results.Statistics.InvalidDocumentCount);
-            Assert.IsNotNull(results.Statistics.TransactionCount);
-            Assert.IsNotNull(results.Statistics.ValidDocumentCount);
+            ValidateBatchDocumentsResult(results, includeStatistics: true);
         }
 
         [Test]
         public async Task RecognizeEntitiesBatchTest()
         {
             TextAnalyticsClient client = GetClient();
-            List<TextDocumentInput> documents = batchDocuments;
+            List<TextDocumentInput> documents = s_batchDocuments;
 
             RecognizeEntitiesResultCollection results = await client.RecognizeEntitiesBatchAsync(documents);
```

Comment:
Could you pass `s_batchDocuments` directly in the `RecognizeEntitiesBatchAsync` method?